R/rules-line-breaks.R

```diff
@@ -236,15 +236,17 @@ remove_line_break_before_round_closing_after_curly <- function(pd) {
 
 remove_line_breaks_in_fun_dec <- function(pd) {
   if (is_function_declaration(pd)) {
-    is_double_indention <- is_double_indent_function_declaration(pd)
+    is_double_indention <- is_single_indent_function_declaration(pd)
```

Comment:
This looks odd. Need to rename `is_double_indention`?

Reply:
@krlmlr can you fix this? You have
```
    is_double_indention <- is_single_indent_function_declaration(pd)
```
I think we need to rename the variable name and keep the function name. Then I'll approve the PR and we can merge.